benchmark-service/benchmark/utils.py

```diff
@@ -14,15 +14,21 @@ def parse_file_path(regex, name):
     return match.groupdict()
 
 
-def enumerate_list_index(list_of_lists):
-    res_list = []
-    i = 0
-    for list in list_of_lists:
-        temp = []
-        temp = [i] * len(list)
-        res_list.extend(temp)
-        i += 1
-    return res_list
+def enumerate_list_index(list_of_trials):
```

Comment:
Sorry, one more nit pick. Let's call this something else because it's no longer generic and expects a certain input. Maybe `enumerate_list_of_trials`.